src/components/v2/appDetails/k8Resource/nodeDetail/NodeDetailTabs/Terminal.component.tsx

```diff
@@ -167,20 +167,20 @@ function TerminalComponent({ selectedTab, isDeleted }) {
                                 minHeight: '24px !important',
                                 cursor: 'pointer',
                             }),
-                            singleValue: (base, state) => ({ ...base, fontWeight: 600, color: '#06c' }),
+                            singleValue: (base, state) => ({ ...base, fontWeight: 600, textAlign: 'left', color: '#06c' }),
                             indicatorsContainer: (provided, state) => ({
                                 ...provided,
                             }),
                             option: (base, state) => ({
-                              ...base,
-                              backgroundColor: state.isFocused ? 'var(--N100)' : 'white',
-                              color: 'var(--N900)',
-                              textOverflow: 'ellipsis',
-                              overflow: 'hidden',
-                              whiteSpace: 'nowrap',
-                              direction: 'rtl',
-                              cursor: 'pointer',
-                          }),
+                                ...base,
```

Comment:
Do we need these option styles as we're overriding them at ReactSelect.util?

Reply:
Removed